src/main/java/org/mule/module/xml/internal/operation/SchemaValidatorOperation.java

```diff
@@ -82,11 +88,15 @@
   @Validator
   @Execution(CPU_INTENSIVE)
   @Throws(SchemaValidatorErrorTypeProvider.class)
-  public void validateSchema(@Path(type = FILE, acceptedFileExtensions = "xsd") String schemas,
+  public void validateSchema(@Optional @Path(type = FILE, acceptedFileExtensions = "xsd") String schemas,
+                             @Optional @NullSafe List<SchemaContent> schemaContents,
                              @Optional(defaultValue = "W3C") SchemaLanguage schemaLanguage,
                              @Content(primary = true) InputStream content,
                              @Config XmlModule config) {
-    withTransformer(new SchemaKey(schemas, schemaLanguage.getLanguageUri(), expandEntities), validator -> {
+    //perform check on schema location field and schema content field, only one can be used and must provided.
```

Comment:
I think is better put this comment where is located the private method, in the line 167 of this class